### PR TITLE
nonce objects and beta content

### PR DESCRIPTION
**Pull request by FofoFromBaba**

### CODE REVIEW

**Comment by someone1065** [comment]
Another green pr. Nice.

**Comment by balt-dev** [comment]
i use github on colorblind mode bc i like the palette more and i was like "huh what's a green pr"

**Comment by balt-dev** [comment]
lemme look around at what this adds

File: data/custom/dizzyandy-sprites.toml [hunk 1]
```diff
@@ -452,3 +452,13 @@ text_metaobj = {sprite = "text_metaobj", color = [0, 2], tiling = "none"}
 
 "text_it'll" = {sprite = "text_it'll", color = [5, 3], tiling = "none"}
 
+text_flora = {sprite = "text_flora", color = [5, 1], tiling = "none"}
+
+text_sway = {sprite = "text_sway", color = [1, 1], tiling = "none"}
+
+text_sustain = {sprite = "text_sustain", color = [2, 4], tiling = "none"}
+
+text_sustain = {sprite = "text_true", color = [0, 3], tiling = "none"}
```
Comment (balt-dev) [inline data/custom/dizzyandy-sprites.toml:461]: this doesn't seem intentional, duplicate keys don't work

**Review by balt-dev — CHANGES_REQUESTED** [review]
noticed something

File: data/custom/dizzyandy-sprites.toml (hunk 1 above)
Comment (someone1065) [reply data/custom/dizzyandy-sprites.toml:461]: You can just pull it and fix it before you restart the bot though, right?

Comment (balt-dev) [reply data/custom/dizzyandy-sprites.toml:461]: i would. rather it be fixed on github.

Comment (someone1065) [reply data/custom/dizzyandy-sprites.toml:461]: I believe you misunderstood. You can pull this and fix it in a separate commit before the changes are applied to the bot itself.

**Review by balt-dev — APPROVED** [review]

**Comment by balt-dev** [comment]
fixed text_true and text_false and also decided to color then (5/4 lime and 2/2 red respectively)

**Pull request merged**